recipes/grpc/all/cmake/grpc_plugin_template.cmake.in

```diff
@@ -5,7 +5,8 @@ if(NOT TARGET @target_name@)
             PATHS ENV PATH
             NO_DEFAULT_PATH
         )
-    else()
+    endif()
+    if(NOT @find_program_variable@)
```

Comment:
Why wouldn't it be not found if cross-compiling? Looks like an issue in dockcross rather than grpc, or how you specify your dependencies in conanfile.

Reply:
@SpaceIm Thanks for your suggestion! I will create a new issue.

Reply:
@SpaceIm https://github.com/conan-io/conan-center-index/issues/13017